db/schema.rb

```diff
@@ -54,8 +54,6 @@
     t.inet     "last_sign_in_ip"
     t.datetime "created_at",                          null: false
     t.datetime "updated_at",                          null: false
-    t.string   "provider"
```

Comment:
We need `provider` and `uid` here, I think this schema.rb was probably based on an old branch.